frontend/src/features/videos/VideoCard.tsx

```diff
@@ -264,12 +238,18 @@ function VideoCard({
   }
 
   return (
-    <Grid container spacing={1} className={classes.main}>
+    <Grid
+      container
+      spacing={1}
+      sx={{
+        ...mainSx(theme),
+      }}
+    >
       <Grid
         item
         xs={12}
         sm={compact ? 12 : 4}
-        style={{ aspectRatio: '16 / 9', padding: 0 }}
+        sx={{ aspectRatio: '16 / 9', padding: 0 }}
```

Comment:
Here too, using `sx` creates a conflict between our custom style, and the defaults defined by MUI.
I am not too fond of using `!important`, but at least it makes explicit that there is a conflict here, and that we may want to update this `<Grid>` layout in the future.

```suggestion
        sx={{ aspectRatio: '16 / 9', padding: '0 !important' }}

```

Reply:
@amatissart Updated the _VideoCard_ component as you suggested.